com.unity.cinemachine/Runtime/Unity.Cinemachine.asmdef

```diff
@@ -76,6 +76,11 @@
             "name": "com.unity.modules.imgui",
             "expression": "1.0.0",
             "define": "CINEMACHINE_UNITY_IMGUI"
```

Comment:
Remove CINEMACHINE_UNITY_IMGUI define, not needed anymore.  Do it in the editor asmdef also